src/iocore/cache/RamCacheLRU.cc

```diff
@@ -107,11 +109,11 @@ RamCacheLRU::resize_hashtable()
   }
   bucket   = new_bucket;
   nbuckets = anbuckets;
-  ats_free(seen);
+  delete seen;
   int size = bucket_sizes[ibuckets] * sizeof(uint16_t);
```

Comment:
Where does `sizeof(uint16_t)` do here? Shouldn't it just be the `bucket_sizes[ibuckets]`?

Reply:
Each bucket stores a uint16_t, no? This is old code.

Reply:
Fixed